grabbit/src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessor.groovy

```diff
@@ -24,12 +24,15 @@ import com.twcable.grabbit.proto.NodeProtos.Properties
 import com.twcable.grabbit.proto.NodeProtos.Property
 import groovy.transform.CompileStatic
```

Comment:
Ah okay.  Yeah, playing a bit in a sandbox, that does seem to be the case.  Maybe I've just had bad luck with edge cases ;)